library/Mockery.php

```diff
@@ -468,7 +468,7 @@ public static function isSame($expected): IsSame
      *
      * @template TMock
      *
-     * @param TMock ...$args
+     * @param array<class-string<TMock>|TMock> $args
```

Comment:
This is invalid type, because according in the doc you can

```
$mock = \Mockery::mock('MyClass', 'MyInterface', [$constructorArg1, $constructorArg2]);
```

Reply:
Yes it is. There are still a lot of different use cases that this does not cover.